azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/utils/Utils.java

```diff
@@ -50,4 +56,29 @@ public static boolean containsIgnoreCase(List<String> list, String str) {
     public static String getId(Object obj) {
         return Integer.toHexString(System.identityHashCode(obj));
     }
+
+    public static <T> T copyProperties(T to, T from) throws IllegalAccessException, InvocationTargetException {
```

Comment:
remove this method and use `org.apache.commons.beanutils.PropertyUtils#copyProperties`